docs/pages/guides/dart/flutter.en-US.mdx

```diff
@@ -44,7 +44,7 @@ You can click the `Use this template` button on the GitHub page to use the templ
 
 ### Method 1: Using Zeabur CLI
 
-First, make sure you have installed the Zeabur CLI according to the [Zeabur CLI deployment guide](../deploy/deploy-in-cli).
+First, make sure you have installed the Zeabur CLI according to the [Zeabur CLI deployment guide](../../deploy/deploy-in-cli).
```

Comment:
It may be better to use the absolute path (the locale will be handled by the docs generator):

```suggestion
First, make sure you have installed the Zeabur CLI according to the [Zeabur CLI deployment guide](/deploy/deploy-in-cli).
```

Reply:
Sounds good, fixed it just now